components/raftstore/src/store/fsm/peer.rs

```diff
@@ -1439,6 +1475,19 @@ where
 
     #[inline]
     fn check_force_leader(&mut self) {
+        if let Some(ForceLeaderState::WaitTicks {
+            failed_stores,
+            ticks,
+        }) = &mut self.fsm.peer.force_leader
+        {
+            *ticks = (*ticks).saturating_sub(1);
```

Comment:
The defense is unnecessary as it's always checked at the very following line.

Reply:
Yes, what if `ticks` is initialized to `0` at first? Though it wouldn't happen in current codebase.

Reply:
Then you can check if it's 0 first and then do the minus. And there is no need to `+1` any more.